app/js/postQuestionsEdit.js

```diff
@@ -0,0 +1,24 @@
+
+// async function postQuestionsEdit (validateQuestionsFormData) {
+async function postQuestionsEdit (obj) {
+    let json = jsonToFormData(obj)
+
+    await fetch('http://localhost:8080/question', {
+        method: 'post',
+        body: json
+    }).then(response => {
+        return response.json()
+    }).then (response => {
+        var responseBox = document.querySelector("#response")
+        if (response.success){
+            responseBox.textContent = response.message
+            responseBox.classList.add("alert-success")
+        } else {
+            responseBox.textContent = "unexpected error, please try again"
+            responseBox.classList.add("alert-danger")
+        }
+    }).catch(error => {
+        responseBox.textContent = "unexpected error, please try again"
+        responseBox.classList.add("alert-danger")
```

Comment:
remove the `alert-success` class in here

Reply:
done